plugins/modules/database_view_info.py

```diff
@@ -0,0 +1,150 @@
+      - prefix1_sys_id: = <some_sys_id>
+        prefix1_atribute_1: = <any_value>
+      - prefix1_sys_id: = <some_sys_id>
+    return_fields:
+      - prefix1_sys_id
+      - prefix1_name
+      - prefix2_name
+      - prefix2_attr_1
+      - prefix2_attr_2
+  register: result
+"""
+
+
+RETURN = r"""
+record:
+  description:
+    - A list of database records.
+    - Note that the fields of the returned records depend on the database view I(database_view) definition.
+    - field names will be prefixed depending on the prefix defined in the database view definition.
+    - be careful each record will have the sum of fields from each single joined table.
+    - use return_fields to limit the fields to be returned.
+  type: list
+  elements: dict
+  returned: success
+  sample:
+    - ms_name: MY_SERVICE
+      ms_sys_id: 762ed2a807e01110befff6fd7c1ed0bf
+      tns_name: AAACDB2-PDB22-DB1
+      tns_operational_status: '1'
+      tns_sys_id: 6a29040107a01110befff6fd7c1ed070
+    - ms_name: MY_SERVICE
+      ms_sys_id: 762ed2a807e01110befff6fd7c1ed0bf
+      tns_name: AAACDB1-PDB11-DB2
+      tns_operational_status: '1'
+      tns_sys_id: 9bb1fee807241110befff6fd7c1ed089
+    - ms_name: MY_SERVICE
+      ms_sys_id: 762ed2a807e01110befff6fd7c1ed0bf
+      tns_name: AAACDB1-PDB12-DB3
+      tns_operational_status: '1'
+      tns_sys_id: fac2be2c07241110befff6fd7c1ed0a9
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+from ..module_utils import arguments, client, errors, query, table
+from ..module_utils.utils import get_mapper
+
+
+def sysparms_query(module, mapper):
+    parsed, err = query.parse_query(module.params["query"] or dict())
+    if err:
+        raise errors.ServiceNowError(err)
+    return query.serialize_query(query.map_query_values(parsed, mapper))
+
+
+def run(module, table_client):
+    database_view = module.params["database_view"]
+    mapper = get_mapper(module, "database_view_item_mapping", dict())
+    query = sysparms_query(module, mapper)
+    q = dict({"sysparm_query": query})
+    if module.params["return_fields"] is not None:
+        q["sysparm_fields"] = ",".join(module.params["return_fields"])
+    return [
+        dict(
+            mapper.to_ansible(record),
+        )
+        for record in table_client.list_records(database_view, q)
+    ]
+
+
+def main():
+    module_args = dict(
+        arguments.get_spec("instance", "query"),
+        database_view=dict(type="str", required=True),
+        return_fields=dict(type="list", elements="str", required=False),
+    )
+
+    module = AnsibleModule(argument_spec=module_args, supports_check_mode=True)
+
+    try:
+        snow_client = client.Client(**module.params["instance"])
+        table_client = table.TableClient(snow_client)
+        records = run(module, table_client)
+        module.exit_json(changed=False, records=records)
+    except errors.ServiceNowError as e:
+        module.fail_json(msg=str(e))
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Please add unit tests for this module. @juremedvesek @tjazsch  I would also want to check if we can use the `api_info` module to achieve this functionality.

Reply:
@Akasurde  @juremedvesek @tjazsch In `api_info` module query parameters are a verbatim copy of what is sent to ServiceNow instance (i.e., the `sysparm_query`). Here, `query` is used as in the other `*_info` modules. Option `return_fields` is named `columns` in other `*_info` modules. My proposal would be to instead add `query` parameter to the `api_info` module, which should be mutually exclusive with the `sysparm_query` parameter, and drop this module.

Reply:
cc @tima

Reply:
Thank you @Akasurde for you review. Is really important for me this PR could be merged.  But I'm sorry,   I'm right now outside enjoying  some holidays  , and I don't have needed tools to  work in this changes . I will continue working on these as soon as I come back.  

Thank you very much again.

Reply:
@Akasurde unit test added